### ISSUE TITLE
Notification of some kind that model / actor scale mis-mash with Motions when not in Meters if importing from Maya.

### ISSUE THREAD

**Issue by shawstar** [o3de/o3de#9436]
**Describe the bug**
Maya has the default working units as Centimeters. Best practices should state we should always be using Meters as a working unit. 

For example, if one artist is working on Modeling in Maya, and another is working on Animation in Maya. But both are using a different Unit set-up, opportunities can come up much like this:

![ScaleDiffCMtoMeter](https://user-images.githubusercontent.com/87207603/167219126-7f6fc6e4-3231-4855-86bd-80f5c546902c.gif)

If possible, notify user when there is a diff scale issue.

**Steps to reproduce**
1. Mix working in Meters and Centimeters when working in DCCs applications.

**Expected behavior**
A notification if scales are mixed in Motion and Actor assets.

**Actual behavior**
No clear idea why an issue is occurring.

**Desktop/Device (please complete the following information):**
 - Device: PC
 - OS: Windows
 - Version 10
 - CPU Intel I9-9900k
 - GPU NVidia RTX 2080ti
 - Memory 16GB

**Additional context**

**Comment by yuyihsu** [comment]
@shawstar can your DCCsi tool fix this issue?

**Comment by amzn-rhhong** [comment]
This is more of a feature request to have a unit convertor for animations. Standard practice for a game studio: You definitely want to agree on a unit before going into modeling / animate.

**Comment by hultonha** [comment]
Agreed, I'm not sure how best to enforce this... is there any metadata that comes in from Maya at import time in the FBX about the units we could scan for and detect? Otherwise I don't know how else to address this other than stating in the docs you should set your DCC tool to use meters to be compatible with O3DE.

**Comment by hultonha** [comment]
@shawstar I wanted to check in to see if this issue is still causing problems and confirm what the severity is. I'll chat with @amzn-rhhong about it when he's back to see if there's some kind of warning we can add to detect this as import time.

**Comment by svitlani** [comment]
@HogJonny-AMZN making sure you have your eyes on this, cause I know you recently worked on DCC tools improvements! Thanks!

**Comment by hultonha** [comment]
Thanks for the update @shawstar. I am hopeful there is some metadata that we can look out for during import that tells us if things have been authored in centimeters or meters and then either adjust (scale) or warn on import. I'm afraid I just don't know enough about it at this time to give a definitive answer. I'll definitely catch-up with @amzn-rhhong when they're back and see what their thoughts are. Also tagging @jackalbe for visibility in case this is something they've come across before. Thanks!

**Comment by jackalbe** [comment]
It is hard to determine the "expected scale" of an incoming 3D asset and even harder to enforce scalar scale across separate assets. For example, a mesh is incoming in units as meters (on asset) and an incoming animation in units as centimeters. 

There are custom user defined properties that can be used to set the scale factor from the FBX format: "UnitScaleFactor" and "OriginalUnitScaleFactor"

**Comment by hultonha** [comment]
Including comment from @AMZN-stankowi from a separate thread

> [potential solution includes]
> - Let's make a new product asset of scene files, an edit-time only file. We don't want to force everyone to generate dbgsg files in all cases, and we wouldn't want to disable the generation of this. We also don't want this information used outside of tool/edit time, it's a waste of space.
> - In the new edit-time only product asset, we can put information about the processed scene that would be useful to other systems. In this case, we could put information about the scale of the file processed.
> - This information would then be available to places like the character tool, which could check the original scale, up-direction, etc of each scene file. It could provide a warning if the scale of the source scene for the animation doesn't match the scale of the source scene for the actor.
>
> Separately, I would also want to investigate why this is happening, I feel like we should automatically fix scale issues like this.

Also @jackalbe shared...

```
/* Check if metadata has information about "UnitScaleFactor" or "OriginalUnitScaleFactor". 
  * This particular metadata is FBX format only. */
if (assImpScene->GetAssImpScene()->mMetaData->HasKey("UnitScaleFactor") ||
    assImpScene->GetAssImpScene()->mMetaData->HasKey("OriginalUnitScaleFactor"))
{
    // If either metadata piece is not available, the default of 1 will be used.
    assImpScene->GetAssImpScene()->mMetaData->Get("UnitScaleFactor", m_unitSizeInMeters);
    assImpScene->GetAssImpScene()->mMetaData->Get("OriginalUnitScaleFactor", m_originalUnitSizeInMeters);

    /* Conversion factor for converting from centimeters to meters.
      * This applies to an FBX format in which the default unit is a centimeter. */
    m_unitSizeInMeters = m_unitSizeInMeters * .01f;
}

// Code\Tools\SceneAPI\SceneBuilder\SceneSystem.cpp
```

Though the above is related to assets that are not FBX assets according to @AMZN-stankowi.

Would definitely be good to discuss more with myself, @amzn-rhhong, @jackalbe and @AMZN-stankowi when possible. Thanks all for the input and helping look into this issue.

**Comment by jackalbe** [comment]
This might lead to some work where the tech reads in the assetinfo file before loading the scene via AssImp or the setting could be plugged in using a scene custom user defined property
